Makefile

```diff
@@ -61,4 +61,7 @@ run-docker:
 exec_app:
 	cd lib/app; bundle _$(shell cat .bundler-version)_ exec ruby console_app.rb
```

Comment:
I guess it makes sense to have `console_app.rb` dedicated for the CLI. I will also change in my PR to let `exec_app` point to `connector.rb`